client/driver/fabfile.py

```diff
@@ -308,11 +308,11 @@ def run_oltpbench_bg():
 
 
 @task
-def run_controller():
+def run_controller(interval_sec=-1):
     LOG.info('Controller config path: %s', dconf.CONTROLLER_CONFIG)
     create_controller_config()
-    cmd = 'gradle run -PappArgs="-c {} -d output/" --no-daemon > {}'.\
-          format(dconf.CONTROLLER_CONFIG, dconf.CONTROLLER_LOG)
+    cmd = 'gradle run -PappArgs="-c {} -t {} -d output/" --no-daemon > {}'.\
+          format(dconf.CONTROLLER_CONFIG, interval_sec, dconf.CONTROLLER_LOG)
```

Comment:
I am just curious how is -t used here? There is no related code in the Java code.

Reply:
https://github.com/cmu-db/ottertune/blob/c31b9f2490abea3398aecc1f06d9a8fd9667777c/client/controller/src/main/java/com/controller/Main.java#L74

Reply:
https://github.com/cmu-db/ottertune/blob/c31b9f2490abea3398aecc1f06d9a8fd9667777c/client/controller/src/main/java/com/controller/Main.java#L101